src/components/banner.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import {
+  Grid,
+  GridContainer,
+  NavDropDownButton,
+} from "@trussworks/react-uswds";
+import { LanguageSelector } from ".";
+import nycLogo from "../images/logos/nyc_logo.png";
+
+function Banner({ children }) {
+  return (
+    <div className="usa-section--dark">
+      <GridContainer>
+        <Grid row  className="banner__logo">
+          <Grid row gap className="banner__logo">
```

Comment:
Does the class name have to be repeated? It's already on line 14.

Reply:
I'll make two class names - good catch there! I'd just wanted to apply flex to both so reused the name